src/app/signalhandler.cpp

```diff
@@ -49,14 +49,11 @@
 #include "base/version.h"
 
 #ifdef STACKTRACE
-#ifdef Q_OS_UNIX
 #include "stacktrace.h"
-#else
-#include "stacktrace_win.h"
-#ifndef DISABLE_GUI
+
+#if defined Q_OS_WIN && !defined DISABLE_GUI
 #include "gui/stacktracedialog.h"
```

Comment:
Can't we display dialog on macOS too?

Reply:
macOS doesn't use dialog currently, see where `StacktraceDialog dlg;` is declared.

However I guess nothing stop us from using stacktrace dialog on all platforms. I'll look into it in a follow up PR.

Reply:
> However I guess nothing stop us from using stacktrace dialog on all platforms.

That's what I meant.


> I'll look into it in a follow up PR.

👍